ClassAttendance/src/WebUI/Identity/UserContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using WebUI.Models.Account;
+
+namespace WebUI.Identity
+{
+    public class UserContext : IdentityDbContext<User>
```

Comment:
I guess you should reuse the existing DbContext in DAL.
It would be easier to support and consistent with all solution architecture.

Reply:
Done